src/modules/FairFormattedOutput.cmake

```
@@ -0,0 +1,56 @@
+################################################################################
+# Copyright (C) 2018-2021 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  #
+#                                                                              #
+#              This software is distributed under the terms of the             #
+#              GNU Lesser General Public Licence (LGPL) version 3,             #
+#                  copied verbatim in the file "LICENSE"                       #
+################################################################################
+
+if(CMAKE_VERSION VERSION_LESS 3.12)
+  message(FATAL_ERROR "Module FairFindPackage2 requires CMake 3.12 or later!")
```

```suggestion
  message(FATAL_ERROR "Module FairFormattedOutput requires CMake 3.12 or later!")
```

We should probably make this dynamic at some point.

---

https://cmake.org/cmake/help/latest/variable/CMAKE_CURRENT_LIST_FILE.html maybe this one works in combination with https://cmake.org/cmake/help/latest/command/get_filename_component.html?

---

BTW: Why are we not using `cmake_minimum_required()`?

---

Updated.